controls/ubuntu-18.04-server-cis-5.2.14.rb

```diff
@@ -50,14 +50,27 @@
   "
   impact 0.5
   tag severity: "medium"
-  tag gtitle: nil
-  tag gid: nil
-  tag rid: nil
-  tag stig_id: nil
-  tag fix_id: nil
-  tag cci: nil
   tag nist: ["SC-8", "SC-8", "Rev_4"]
   tag cis_level: 1
   tag cis_controls: ["14.4", "16.5", "Rev_7"]
   tag cis_rid: "5.2.14"
+  cfg = parse_config_file('/etc/ssh/sshd_config', { assignment_regex: /^\s*(\S*)\s*(.*?)\s*$/ } )
+  describe cfg do
+    its('MACs') { should be }
```

Comment:
should be ?? ... what

Reply:
It should exist (ie, be truthy, or just be.... in a metaphysical sense :) ): https://relishapp.com/rspec/rspec-expectations/v/3-10/docs/built-in-matchers/be-matchers#be-matcher

This ensures that the list of MACs is set, and doesn't rely on default values (IIUC, the default value would allow forbidden weak algorithms).